plugins/libvirt/kvm_io

```diff
@@ -11,12 +11,15 @@
 # parsed environment variables:
 # vmsuffix: part of vm name to be removed
 #
-#%# capabilities=autoconf
-#%# family=contrib
+# %# capabilities=autoconf
```

Comment:
The missing space here is crucial for matching these magic tokens :)

Please use the opportunity for turning the documentation into the perlpod format (see other plugins for examples).